colin/core/check_runner.py

```diff
@@ -41,13 +39,13 @@ def _result_generator(target, checks, timeout=None):
                 logger.debug("Check timeout: {}".format(_timeout))
                 yield exit_after(_timeout)(check.check)(target)
             except TimeoutError as ex:
-                logger.warning(
-                    "The check hit the timeout: {}".format(_timeout))
+                logger.warning("The check hit the timeout: {}".format(_timeout))
```

Comment:
I assume it was automatically re-formated but 
`string.format` should not be use in logger :-)

And there are few other similar places.